integration/transition-test.ts

```diff
@@ -190,6 +202,17 @@ describe("rendering", () => {
     expect(html).toMatch(PAGE_INDEX_TEXT);
   });
 
+  test("loader redirect with hash", async () => {
+    await app.goto("/");
+
+    let responses = app.collectDataResponses();
```

Comment:
It seems like the response isn't used, so we could just do the call without assigning it I guess

```suggestion
    app.collectDataResponses();
```

Reply:
That was a copy/paste leftover, thanks for noticing !